rai/node/node.cpp

```diff
@@ -1899,6 +1953,20 @@ void rai::node::ongoing_rep_crawl ()
 	}
 }
 
+void rai::node::ongoing_rep_calculation ()
+{
+	auto now (std::chrono::steady_clock::now ());
+	vote_processor.calculate_weights ();
+	std::weak_ptr<rai::node> node_w (shared_from_this ());
+	// Repeat in 10 minutes
+	alarm.add (now + std::chrono::seconds (10 * 60), [node_w]() {
```

Comment:
Minor: maybe `std::chrono::minutes` here, and the comment can be removed?

Reply:
Sure